pkg/reconciler/workload/apiexport/apiresourceschema.go

```diff
@@ -72,3 +75,12 @@ func toAPIResourceSchema(r *apiresourcev1alpha1.NegotiatedAPIResource, name stri
 
 	return schema
 }
+
+// ParseAPIResourceSchemaName parses name of APIResourceSchema to a gr and schema if it is valid.
+func ParseAPIResourceSchemaName(name string) (schema.GroupResource, string, bool) {
```

Comment:
There are no users of the second `string` this returns - can we just omit it until we need it?